src/style.css

```diff
@@ -0,0 +1,121 @@
+html {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+}
+
+.card {
```

Comment:
checklist 10th item, [FILE STRUCTURE] - You should create separate file for each BEM block